src/lib/is_tap.js

```diff
@@ -0,0 +1,16 @@
+const euclideanDistance = require('./euclidean_distance');
+
+const TOLERANCE = 25;
+const INTERVAL = 250;
+
+module.exports = function isTap(start, end, options = {}) {
```

Comment:
@ChrisBellew - what are your thoughts on using `is_click` to do this check? The code is very similar here. Also, any notes on why we're using `250` and `25` for the interval and tolerance here? The `is_click` values is from experience from the iD editor.